docs/_articles/en/vscode-desktop/recommended-extensions.md

```diff
@@ -1,41 +1,49 @@
 ---
-title: Recommended Extensions
+title: Other Recommended Extensions
 lang: en
 ---
 
+#  Extensions We Recommend
 In addition to the extension pack shipped by Salesforce, the following extensions are useful for Salesforce developers. We also package the Extension pack with some of the most-used community extensions in the [expanded extension pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-expanded).
 
+> **_NOTE:_** Because Code Builder is a web-based and comes with pre-installed extensions, this list is not of relevance if Code Builder is your development environment of choice.
```

Comment:
This explanation is wordy. Can you pare it down? Is this a beta limitation or the way it'll always be for CB? 
I would word this differently depending on if we plan to allow other extensions in the future. If not:
You can install more extensions only if using Salesforce Extensions for VS Code on your desktop. Code Builder comes fully configured so you can't install additional software (or just extensions).